src/System.Private.CoreLib/shared/System/Threading/Timer.cs

```diff
@@ -124,7 +124,7 @@ private bool EnsureTimerFiresBy(uint requestedDuration)
         private const int ShortTimersThresholdMilliseconds = 333;
 
         // Time when Pause was called
-        private volatile int _pauseTicks = 0;
+        private readonly int _pauseTicks = 0;
```

Comment:
Shouldn't this be `const` instead?

Reply:
I'm actually going to follow this up with a separate change that deletes _pauseTicks entirely.